tools/src/tools.ml

```diff
@@ -676,3 +676,183 @@ let extractEmbedded ~extensionPoints ~filename =
              ("loc", Some (Analysis.Utils.cmtLocToRange loc |> stringifyRange));
            ])
   |> List.rev |> array
+
+module FormatDocstrings = struct
+  let mapRescriptCodeBlocks ~colIndent ~(mapper : string -> int -> string)
+      (doc : string) =
+    let indent = String.make colIndent ' ' in
+    let len = String.length doc in
+    let buf = Buffer.create len in
+    let addIndent () = Buffer.add_string buf indent in
+    let currentCodeBlockContents = ref None in
+    let lines = String.split_on_char '\n' doc in
+    let lineCount = ref (-1) in
+    let rec processLines lines =
+      let currentLine = !lineCount in
+      lineCount := currentLine + 1;
+      match (lines, !currentCodeBlockContents) with
+      | l :: rest, None ->
+        if String.trim l = "```rescript" then (
```

Comment:
Maybe check with a lowercase compare?
Allowing `ReScript` as well.

Reply:
Lowercase language identifier is the standard for tagging md code blocks. But we should probably add support for ```res as well.

Reply:
Feels more like the thing you encountered the most in the wild rather than a hard standard.
https://github.github.com/gfm/#info-string does not mention casing.